testing/integration-tests/src/full_client/mod.rs

```diff
@@ -7,6 +7,8 @@ mod blocks;
 #[cfg(test)]
 mod client;
 #[cfg(test)]
+mod codegen;
```

Comment:
we can probably use in #![cfg(test)] in globally here instead repeating test for each module :)